### PR TITLE
[WEB-163] Benefits section

### PR DESCRIPTION
**Pull request by snksbs**

### CODE REVIEW

File: web/components/RenderSections.js [hunk 1]
```diff
@@ -72,6 +72,28 @@ function RenderSections(props) {
...
+      if (section.route) {
+        for (let i = 0; i < routes.length; i++) {
+          const route = routes[i]
+
+          if (route._id === section.route._ref) {
+            section.route = route // TODO Add break later and verify if breaks current functionality somehow
+          }
+        }
+      }
+    }
```
Comment (davidverity) [inline web/components/RenderSections.js:96]: Why did you add the route validation here?

File: web/components/RenderSections.js [hunk 2]
```diff
@@ -72,6 +72,28 @@ function RenderSections(props) {
         }
       }
     }
+    if (section.benefits && section.benefits[0]._ref) {
```
Comment (davidverity) [inline web/components/RenderSections.js:75]: Jump a line before the if, please

File: studio/schemas/documents/sections/keyBenefits.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
+import { MasterDetailIcon } from '@sanity/icons'
+import supportedLanguages from '../../supportedLanguages';
+
+
```
Comment (davidverity) [inline studio/schemas/documents/sections/keyBenefits.js:4]: Remove this extra line, please

File: studio/schemas/documents/benefitCard.js [hunk 4]
```diff
@@ -0,0 +1,43 @@
+import { DocumentIcon } from '@sanity/icons'
+import supportedLanguages from '../supportedLanguages';
+
+
```
Comment (davidverity) [inline studio/schemas/documents/benefitCard.js:4]: Remove this extra line, please

File: web/components/sections/index.js [hunk 5]
```diff
@@ -4,6 +4,7 @@ export {default as HeroDoubleButton} from './HeroDoubleButton/HeroDoubleButton'
 export {default as HeroWithImage} from './HeroWithImage'
 export {default as Timeline} from './Timeline/Timeline'
 export {default as ImageBesideText} from './ImageBesideText'
+export { default as KeyBenefits } from './KeyBenefits'
```
Comment (davidverity) [inline web/components/sections/index.js:7]: Can you please adjust it without the spaces between the {}?

File: web/pages/[[...slug]].js [hunk 6]
```diff
@@ -1,13 +1,13 @@
 import imageUrlBuilder from '@sanity/image-url'
 import groq from 'groq'
-import {NextSeo} from 'next-seo'
-import {useRouter} from 'next/router'
+import { NextSeo } from 'next-seo'
```
Comment (davidverity) [inline web/pages/[[...slug]].js:3]: Please, revert the indentationchanges not related to this PR

**Review by davidverity — CHANGES_REQUESTED** [review]
I've left a few coments on each code section, please take a look and let me know if there are any doubts

File: studio/schemas/documents/sections/keyBenefits.js [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+      title: `title.${baseLanguage.id}`,
+      // firstBenefitHeading: `benefit.0.heading`,
+      // secondBenefitHeading: `benefit.1.heading`,
+      // thirdBenefitHeading: `benefit.2.heading`,
+      // fourthBenefitHeading: `benefit.3.heading`,
+    },
+    prepare({ title = 'No title' }) {
+      return {
+        title,
+      };
```
Comment (davidverity) [inline studio/schemas/documents/sections/keyBenefits.js:42]: Please add a subtitle:

subtitle: 'Key Benefits section'

File: studio/schemas/documents/sections/keyBenefits.js [hunk 8]
```diff
@@ -0,0 +1,45 @@
+import { MasterDetailIcon } from '@sanity/icons'
```
Comment (davidverity) [inline studio/schemas/documents/sections/keyBenefits.js:1]: Please, change the icon to SplitHorizontalIcon

File: studio/schemas/documents/sections/keyBenefits.js [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+import supportedLanguages from '../../supportedLanguages';
+
+
+const baseLanguage = supportedLanguages.find(l => l.isDefault);
+
+export default {
+  name: 'keyBenefits',
+  type: 'document',
+  title: 'Key Benefits',
+  icon: MasterDetailIcon,
```
Comment (davidverity) [inline studio/schemas/documents/sections/keyBenefits.js:11]: Please, change the icon to SplitHorizontalIcon

File: web/components/RenderSections.js (hunk 1 above)
Comment (snksbs) [reply web/components/RenderSections.js:87]: removed

File: web/components/RenderSections.js (hunk 2 above)
Comment (snksbs) [reply web/components/RenderSections.js:76]: done

File: studio/schemas/documents/sections/keyBenefits.js (hunk 7 above)
Comment (snksbs) [reply studio/schemas/documents/sections/keyBenefits.js:41]: done

File: studio/schemas/documents/sections/keyBenefits.js (hunk 9 above)
Comment (snksbs) [reply studio/schemas/documents/sections/keyBenefits.js:10]: done

**Review by davidverity — APPROVED** [review]
Looks amazing, well done 
image
